packages/sdk/tsconfig.json

```diff
@@ -1,17 +1,21 @@
 {
   "compilerOptions": {
     "strict": true,
-    "target": "es5",
-    "module": "commonjs",
+    "target": "es2017",
```

Comment:
Btw something to clarify, the sdk is meant to be also usable in the frontend, hence the `es5` target :)